src/App.js

```diff
@@ -1,5 +1,127 @@
+const MissionUtils = require("@woowacourse/mission-utils");
+
 class App {
-  play() {}
+  play() {
+    let computer = [];
+    computer = computer_randomnumber();
```

Comment:
airbnb 네이밍 컨벤션에 따라 snake_case 보단 camelCase 를 사용하는게 좋아보입니다.
```suggestion
    computer = computerRandomnumber();
```

Reply:
아앗 .. 조언 감사합니다 !!!!